webapp/components/search_bar.jsx

```diff
@@ -270,15 +274,6 @@ export default class SearchBar extends React.Component {
                 >
                     <span className='fa fa-angle-left'/>
                 </div>
-                <span
-                    className='search__clear'
-                    onClick={this.clearFocus}
-                >
-                    <FormattedMessage
-                        id='search_bar.cancel'
```

Comment:
'search_bar.cancel' - this is not being used elsewhere. Aside from en, should I delete it as well from other locale?